helpers/db.py

```diff
@@ -0,0 +1,57 @@
+import lancedb
+from helpers.models import TextChunk, Document
+from typing import Iterable, Literal
+from pathlib import Path
+
+
+def create_db(path):
```

Comment:
we dont need functions that just call the a function 

def f(a):
   return g(a)